.buildkite/pipeline.yml

```diff
@@ -82,12 +82,17 @@ steps:
       DOCKER_IMAGE_TAG: "$BUILDKITE_BUILD_NUMBER"
       NUMBER_OF_VMS: 10
       EXTRAGOARGS: "-v -count=1 -race"
+      FICD_DM_VOLUME_GROUP: fcci-vg
     artifact_paths:
-      - "snapshotter/logs/*"
+      - "runtime/logs/*"
     command:
-      - make -C snapshotter integ-test
+      - make -C runtime integ-test FICD_DM_POOL=build_${BUILDKITE_BUILD_NUMBER}_runtime
```

Comment:
do we need a devmapper pool for these tests? If so, might want to reset it before running a la 	https://github.com/firecracker-microvm/firecracker-containerd/blob/47ceb625562fadcd7d615f519a24086a45cf73cc/runtime/Makefile#L54-L55